src/python/grpcio_tests/tests_aio/benchmark/worker_servicer.py

```diff
@@ -0,0 +1,368 @@
+                grpc.unary_unary_rpc_method_handler(servicer.UnaryCall),
+        }
+        handler = grpc.method_handlers_generic_handler(
+            'grpc.testing.BenchmarkService', method_implementations)
+        server.add_generic_rpc_handlers((handler,))
+    else:
+        raise NotImplementedError('Unsupported server type {}'.format(
+            config.server_type))
+
+    if config.HasField('security_params'):  # Use SSL
+        server_creds = grpc.ssl_server_credentials(
+            ((resources.private_key(), resources.certificate_chain()),))
+        port = server.add_secure_port('[::]:{}'.format(config.port),
+                                      server_creds)
+    else:
+        port = server.add_insecure_port('[::]:{}'.format(config.port))
+
+    return server, port
+
+
+def _get_client_status(start_time: float, end_time: float,
+                       qps_data: histogram.Histogram
+                      ) -> control_pb2.ClientStatus:
+    """Creates ClientStatus proto message."""
+    latencies = qps_data.get_data()
+    end_time = time.time()
+    elapsed_time = end_time - start_time
+    stats = stats_pb2.ClientStats(latencies=latencies,
+                                  time_elapsed=elapsed_time,
+                                  time_user=elapsed_time,
+                                  time_system=elapsed_time)
+    return control_pb2.ClientStatus(stats=stats)
+
+
+def _create_client(server: str, config: control_pb2.ClientConfig,
+                   qps_data: histogram.Histogram
+                  ) -> benchmark_client.BenchmarkClient:
+    """Creates a client object according to the ClientConfig."""
+    if config.load_params.WhichOneof('load') != 'closed_loop':
+        raise NotImplementedError(
+            f'Unsupported load parameter {config.load_params}')
+
+    if config.client_type == control_pb2.ASYNC_CLIENT:
+        if config.rpc_type == control_pb2.UNARY:
+            client_type = benchmark_client.UnaryAsyncBenchmarkClient
+        elif config.rpc_type == control_pb2.STREAMING:
+            client_type = benchmark_client.StreamingAsyncBenchmarkClient
+        else:
+            raise NotImplementedError(
+                f'Unsupported rpc_type [{config.rpc_type}]')
+    else:
+        raise NotImplementedError(
+            f'Unsupported client type {config.client_type}')
+
+    return client_type(server, config, qps_data)
+
+
+def _pick_an_unused_port() -> int:
+    """Picks an unused TCP port."""
+    _, port, sock = get_socket()
+    sock.close()
+    return port
+
+
+async def _create_sub_worker() -> _SubWorker:
+    """Creates a child qps worker as a subprocess."""
+    port = _pick_an_unused_port()
+
+    _LOGGER.info('Creating sub worker at port [%d]...', port)
+    process = await asyncio.create_subprocess_exec(sys.executable,
+                                                   _WORKER_ENTRY_FILE,
+                                                   '--driver_port', str(port))
```

Comment:
Maybe I'm playing as the advocate of the devil, but IMO there is an outstanding question.

How much could benefit the synchronous client by parallelizing also by using different processes? 

Having the feeling that the QPS of the Synchronous client once we start parallelizing by a process, besides by threads within each process, would bost up the QPS.